docs/data/joy/getting-started/usage/usage.md

```diff
@@ -1,42 +1,109 @@
 # Usage
 
-<p class="description">Learn the basics of using Joy UI components in two quick steps.</p>
+<p class="description">Learn the basics of using Joy UI components.</p>
 
 ## Getting started
 
-### Set up the `CssVarsProvider` component
-
-Go to your `App.js` file and replace it with the code snippet below.
-
-You should see the text `Hello from Joy` being rendered on your browser.
+The following code snippet demonstrates a simple app that uses the Joy UI [Button](/joy-ui/react-button/) component:
 
 ```jsx
+import * as React from 'react';
 import { CssVarsProvider } from '@mui/joy/styles';
+import Button from '@mui/joy/Button';
 
-function App() {
-  return <CssVarsProvider>Hello from Joy</CssVarsProvider>;
+export default function App() {
+  return (
+    <CssVarsProvider>
+      <Button>Hello World</Button>
+    </CssVarsProvider>
+  );
 }
+```
+
+### CssVarsProvider
+
+In the example above, you can see that the Button component is nested within `<CssVarsProvider />`.
+This is not required, but it unlocks a whole host of customization options powered by CSS variables.
+See [Using CSS variables](/joy-ui/customization/using-css-variables/) for more details.
+
+## Shared props
+
+Joy UI components are self-supporting and fully functional in isolation.
 
-export default App;
+Each component has its own unique API, but all components accept the following shared props:
+
+### components
```

Comment:
Although, this is a very useful section I feel that it should be on a different page that explains "slot overrides".

To me, the usage page should be simple as possible to start using components.

I believe that the slot overrides are not common use cases and they should be in an advanced guide.

Reply:
Agreed. We should probably mention somewhere that everything as far as these shared props goes are exactly the same as Base, since it is using Base components under the hood anyway.

Reply:
That's interesting feedback. MUI Base has a 4th page in the "Getting started" sequence after Usage called [Customization](https://mui.com/base/getting-started/customization/). Maybe this page would be a better place to document these more advanced props?

Reply:
Agreed.

Reply:
Yeah I think that makes sense. In that case, I'll remove that info from these pages in this PR, and create a new PR to standardize the "Customization" pages and add one for Joy. Does that sound good?

Reply:
Sounds good!